client/src/app/site/common/components/start/start.component.ts

```diff
@@ -142,7 +130,7 @@ export class StartComponent extends BaseComponent implements OnInit {
     }
 
     public test(): void {
```

Comment:
I do not need these test anymore. Can you remove it completly (incl. the button in the ui)?